reconcile/utils/saasherder/models.py

```diff
@@ -113,14 +113,15 @@ def state_key(self) -> str:
 
 @dataclass
 class TriggerSpecContainerImage(TriggerSpecBase):
-    image: str
+    images: Sequence[str]
     reason: str | None = None
 
     @property
     def state_key(self) -> str:
+        image_key = ",".join(self.images)
```

Comment:
should `images` be sorted to avoid order change affect state key?

Reply:
Good catch! Fixed

Reply:
`images` declared twice, `images: Sequence[str]` and `@property` both set it, can just sort on the fly inside `def state_key`, shouldn't be slow since usually small images, and you changed separator from `,` to `/`?

Reply:
oh, that duplicate already fixed, just confirm about `,` vs `/` then

Reply:
Hmm I think you are on an older commit. Most recent shouldnt have property anylonger

Reply:
Yeah `/` vs `,` is intentional